tests/mtouch/MTouchTool.cs

```diff
@@ -326,14 +326,79 @@ public IEnumerable<DeviceInfo> FindAvailableDevices (string [] deviceClasses)
 			}
 		}
 
-		public void CreateTemporaryApp (string code = null)
+		string CreatePlist (MTouch.Profile profile, string appName)
+		{
+			string plist = null;
+
+			switch (profile) {
+			case MTouch.Profile.Unified:
+				plist = string.Format (@"<?xml version=""1.0"" encoding=""UTF-8""?>
+<!DOCTYPE plist PUBLIC ""-//Apple//DTD PLIST 1.0//EN"" ""http://www.apple.com/DTDs/PropertyList-1.0.dtd"">
+<plist version=""1.0"">
+<dict>
+	<key>CFBundleDisplayName</key>
+	<string>{0}</string>
+	<key>CFBundleIdentifier</key>
+	<string>com.xamarin.{0}</string>
+	<key>CFBundleExecutable</key>
+	<string>{0}</string>
+	<key>MinimumOSVersion</key>
+	<string>6.0</string>
```

Comment:
There's a better way than hardcoding versions:  there's a method that will get you the latest version given a profile, just fill in with that (same for the min tvOS version below).